site/_docs/themes.md

```diff
@@ -14,7 +14,11 @@ Jekyll has an extensive theme system, which allows you to leverage community-mai
 
 2. Save the changes to your `Gemfile`
 3. Run the command `bundle install` to install the theme
-4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+4. Set your `/css/main.scss` to import your theme partials using the `@import` directive:
+
+        @import: my-awesome-jekyll-theme
```

Comment:
@parkr sorry, I did not mean to include the colon. must've been half-asleep.. :grin: 

@DirtyF, If you check the full page, you'll see that `my-awesome-jekyll-theme` was the example in the steps preceding this.
I'll update after you confirm.